packages/opentelemetry-exporter-collector/README.md

```diff
@@ -38,7 +38,7 @@ const { CollectorExporter } =  require('@opentelemetry/exporter-collector');
 
 const collectorOptions = {
   serviceName: 'basic-service',
-  url: '<opentelemetry-collector-url>' // url is optional and can be omitted - default is http://localhost:55678/v1/trace
+  url: '<opentelemetry-collector-url>' // url is optional and can be omitted - default is http://localhost:55678
```

Comment:
I would mention both (Node and Web) default URLs.

Reply:
@mayurkale22 Should I also throw an error/alter the URL when someone enters an incorrect URL? (e.g. entering a URL that ends in `/v1/trace` for Node)